web/src/components/Card/styles.js

```diff
@@ -0,0 +1,9 @@
+import { makeStyles } from "@mui/material";
```

Comment:
Are we importing these styles anywhere?

Reply:
It could have been used to add styles to the card but we are going to remove mui so I will remove the file in the coming commit.